docs/reference/cli-commands/notice.md

```diff
@@ -18,3 +18,38 @@ by unique type and key combination (2-arg variant).
       --uid=            Look up notice from user with this UID (admin only; 2-arg variant only)
 ```
 <!-- END AUTOMATED OUTPUT -->
+
+## Examples
+
+You can fetch a notice either by ID or by type/key combination. To fetch the notice with ID "1":
+
+```{terminal}
+   :input: pebble notice 1
+id: "1"
+user-id: 1000
+type: custom
+key: example.com/foo
+first-occurred: 2023-09-15T04:16:09.179395298Z
+last-occurred: 2023-09-15T04:16:19.487035209Z
+last-repeated: 2023-09-15T04:16:09.179395298Z
+occurrences: 3
+expire-after: 168h0m0s
+```
+
+To fetch the notice with type "custom" and key "other.com/bar":
```

Comment:
"other.com/bar" seems to be getting rendered as a link.
I think you need to escape it with a <span></span> tag.

https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide-myst/#external-links